src/urls.js

```diff
@@ -51,6 +51,18 @@ class Urls {
     return `${this.groupsBase()}/${id}`;
   }
 
+  getGroupActivity(id, query) {
+    return `${this.group(id)}/activity${this.queryString(query)}`;
+  }
+
+  getGroupAdmins(id) {
+    return `${this.groupsBase()}/${id}/admins`;
+  }
+
+  getGroupMembers(id) {
+    return `${this.groupsBase()}/${id}/members`;
```

Comment:
`getGroupMembers` should also accept a query string (source: https://github.com/vinyldns/vinyldns/blob/44eed12b08510750a7688714159be9e6d8319ed9/modules/api/src/main/scala/vinyldns/api/route/MembershipRouting.scala#L124-L127).

Reply:
Thanks! I overlooked that.